tests/aggregation/tests.py

```diff
@@ -988,6 +988,30 @@ def test_annotated_aggregate_over_annotated_aggregate(self):
         with six.assertRaisesRegex(self, FieldError, "Cannot compute Sum\('id__max'\): 'id__max' is an aggregate"):
             Book.objects.annotate(Max('id')).annotate(Sum('id__max'))
 
+        class MyMax(Max):
+            def as_sql(self, compiler, connection):
+                self.set_source_expressions(
+                    self.get_source_expressions()[0:1])
+                return super(MyMax, self).as_sql(compiler, connection)
+
+        with six.assertRaisesRegex(self, FieldError, "Cannot compute Max\(...'id__max'...\): 'id__max' is an aggregate"):
```

Comment:
omitting the ellipsis seems okay to me. Do we need to include 'id__max' twice, e.g. `Cannot compute Max(): 'id__max' is an aggregate`